ports/bitsery/portfile.cmake

```
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO fraillt/bitsery
-    REF c0fc083c9de805e5825d7553507569febf6a6f93 # v5.2.2
-    SHA512 a4c8660f6e8dcb5162f6f75e0f1e4716032b8403e9461f42e0628955eb07dc7c17aec9f774f45c2c15cce28a231699a71815d3d6d7d0f34a1367ee1e2d944305
+    REF "v${VERSION}"
+    SHA512 59af7fc4b3647703fe228a7fd2a23e454e0fa42c849f718e5a731c34e1e427bc481d4ae6909f55682daf61a8d3ea5a1e37a6cf1538591a132b6b306acf06f872
```

```suggestion
    SHA512 6edaf4d5f22d640df67813f7525c2e076a76911079e25eeb93f4302a835b26d986379d1aad6bdb98f1781af3a5d638ee02ea9c32765741ce91366b518fc4d447
```
For new functions, a new hash is required.